helm/values.yaml

```diff
@@ -0,0 +1,166 @@
+image: "registry.scontain.com/scontain/raisson/pccs:1.21"
+replicas: 1
+
+imagePullSecrets:
+  - name: pccs-pull-secret
+
+# Container-level security settings to minimize security risks
+securityContext:
+  runAsUser: 999
+  fsGroup: 999
+  runAsNonRoot: true
+
+# Startup probe checks if the application has successfully started
+startupProbe:
+  enabled: true
+  path: "/sgx/certification/v4/rootcacrl"
+
+# Liveness probe ensures the application is still running and responsive
+livenessProbe:
+  enabled: true
+  path: "/sgx/certification/v4/rootcacrl"
+
+# Readiness probe ensures the service is ready to accept traffic
+readinessProbe:
+  enabled: true
+  path: "/sgx/certification/v4/rootcacrl"
+
+# Kubernetes service configuration
+service:
+  port: 8665
+
+# Persistent Volume Claim (PVC) Configuration
+persistentVolumeClaim:
+
+  # Access modes options: ReadWriteOnce, ReadOnlyMany and ReadWriteMany
+  accessModes:
+    - ReadWriteOnce
+
+  # Volume size (e.g., 5Gi, 500Mi)
+  size: "1Gi"
+
+  # Storage class for PVC (e.g., "standard", "default")
+  storageClassName: "default"
+
+pccsConfig:
+  # The hosts that will be accepted for connections. Default is localhost only. To accept all connections use 0.0.0.0
+  host: "0.0.0.0"
+
+  # The URL of Intel Provisioning Certificate Service. The default URL is https://api.trustedservices.intel.com/sgx/certification/v4/
+  uri: "https://api.trustedservices.intel.com/sgx/certification/v4/"
+
+  # The PCCS uses this API key to request collaterals from Intel's Provisioning Certificate Service. User needs 
+  # to subscribe first to obtain an API key. For how to subscribe to Intel Provisioning Certificate Service and
+  # receive an API key, goto https://api.portal.trustedservices.intel.com/provisioning-certification and click
+  # on 'Subscribe'.
+  apiKey: "your-api-key"
```

Comment:
Leave it empty, and make it required in HELM template. In this way, you avoid misconfiguration using this string value.